framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -636,6 +636,16 @@ private AnnotationMirror createBooleanAnnotationMirror(List<Boolean> values) {
     }
 
     @Override
+    public List<AnnotationMirror> getJaifInferenceIgnoredAnnotations() {
+        List<AnnotationMirror> out = super.getJaifInferenceIgnoredAnnotations();
+        out.add(AnnotationUtils.fromClass(analysis.getEnv().getElementUtils(),
+                UnknownVal.class));
```

Comment:
How do we know that these values are part of the type hierarchy?  Would this be better to make each type system do individually?  Also, what is the criterion for being added?  Is it any annotation that should not be written by a programmer, or something else?  Documenting that would be good, and then you could also make the logic more specific.